site/src/App/routes/gallery/Gallery.tsx

```diff
@@ -136,43 +137,52 @@ const getRowsFor = memoize((type: SetName) => {
 interface RenderExampleProps {
   id: string;
   example: ComponentExample;
+  isAnIcon: boolean;
```

Comment:
Even though this was an existing variable before your change, while you're here IMO it should be renamed to `isIcon`:
```suggestion
  isIcon: boolean;
```